### DISCUSSION TITLE
make the combo count have a different color if you're fcing the map

### DISCUSSION THREAD

**Discussion by Faustas156** [ppy/osu#16534] [Ideas] [answered]
the title says it all, i'd personally like to see a color on the bottom left side where your combo is to be a different color indicating if you're FCing the map, if you break anywhere during the song, the combo color will return back to a normal white combo color. this is just more of a QoL feature that can be turned on/off.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/8413
